### ISSUE TITLE
Use Stan models instead of GeMTC

### ISSUE THREAD

**Issue by epiben** [OHDSI/NetworkMetaAnalysis#1]
To have a functional package up and running fairly quickly, we use the GeMTC package because it has a simple API that play well with our needs. GeMTC is uses JAGS for sampling, which means that each have to be built and compiled for every network metaanalysis. With Stan, we could write and compile the models once (significant pre-sampling speed-up), and because Stan uses the No-U-turn-sampler (NUTS) variant of Hamiltonian Monte Carlo sampling sampling would likely yield speed-up during sampling. Further, the efficiency of the NUTS sampler should require fewer posterior draws to yield the same number of efficient samples (another speed-up). 

In short, we should write the appropriate Stan models and change the package to play with the results of those.

**Comment by epiben** [comment]
Started porting the model, but the NUTS sampler of Stan seems more picky than the Gibbs sampler of WinBUGS/JAGS, so still some way to go before this is ready. We'll stick with the JAGS models of GeMTC for now.
